src/dotnet-install.ps1

```diff
@@ -1170,6 +1177,12 @@ function Prepare-Install-Directory {
     }
 }
 
+if ($Help)
+{
+    Get-Help $PSCommandPath -Examples
+    exit
+}
```

Comment:
Thank you @baronfel for the PR! Will it make sense to cover those cases by tests to verify that the output is the one that we expect? Also test coverage will allow us to catch changes of help method when they are not expected :) 
What do you think?

Reply:
I added some basic tests - we don't have any coverage of this currently so this will at least pin the current state. I don't know why the jobs got stuck, though.

Reply:
Thank you! Currently there is an ongoing PR: https://github.com/dotnet/install-scripts/pull/411 with the updated tests which should solve this.

Reply:
I have updated the tests. 
the problem mainly was in the verify command, since it requires to have something to compare with which was not added at that point. However I changed the approach to use explicit comparison of some rows of the ouput because the all help output could differ from machine to machine depending on the configuration.

Reply:
Thank you @f-alizada!